runelite-client/src/main/java/net/runelite/client/plugins/worldmap/QuestStartLocation.java

```diff
@@ -46,7 +46,8 @@
 	THE_RESTLESS_GHOST("The Restless Ghost", new WorldPoint(3240, 3210, 0)),
 	RUNE_MYSTERIES("Rune Mysteries", new WorldPoint(3210, 3220, 0)),
 	SHEEP_SHEARER("Sheep Shearer", new WorldPoint(3190, 3272, 0)),
-	SHIELD_OF_ARRAV("Shield of Arrav", new WorldPoint(3208, 3495, 0)),
+	SHIELD_OF_ARRAV_1("Shield of Arrav (Phoenix Gang)", new WorldPoint(3208, 3495, 0)),
```

Comment:
Better enum names would be helpful here. `SHIELD_OF_ARRAV_PHOENIX_GANG` and `SHIELD_OF_ARRAV_BLACK_ARM_GANG` maybe?